ush/setup.py

```diff
@@ -778,6 +791,23 @@ def _remove_tag(tasks, tag):
            "metatask_PointStat_SFC_UPA_all_mems",
            "metatask_GenEnsProd_EnsembleStat_SFC_UPA",
            "metatask_PointStat_SFC_UPA_ensmeanprob"]
+
+        vx_field_groups_all_by_obtype["AERONET"] = ["AOD"]
+        vx_metatasks_all_by_obtype["AERONET"] \
+        = ["task_get_obs_aeronet",
+           "metatask_ASCII2nc_obs"]
+
+        vx_field_groups_all_by_obtype["AIRNOW"] = ["PM25", "PM10"]
+        vx_metatasks_all_by_obtype["AIRNOW"] \
+        = ["task_get_obs_airnow",
+           "metatask_ASCII2nc_obs"
+           "metatask_PcpCombine_fcst_PM_all_mems"]
+
+        vx_field_groups_all_by_obtype["AERONET"] = ["AOD"]
+        vx_metatasks_all_by_obtype["AERONET"] = ["task_get_obs_aeronet","metatask_ASCII2nc_obs"]
+
+        vx_field_groups_all_by_obtype["AIRNOW"] = ["PM25","PM10"]
+        vx_metatasks_all_by_obtype["AIRNOW"] = ["task_get_obs_airnow","metatask_ASCII2nc_obs","metatask_PcpCombine_fcst_PM_all_mems"]
```

Comment:
@mkavulich Looks like the first two sections are repeated in sections 3 and 4 except with everything on a single line.  I guess remove the first two or last two stanzas/sections.

Reply:
Thanks for catching this, this was just a bad merge resolution. Fixed.